src/types/types.ts

```diff
@@ -237,23 +237,26 @@ export interface PageInfo {
   rowsOnCurrentPage: number;
 }
 
-export type CellStyling = {
-  fontSize: string;
-  fontFamily: string;
+export type BasicCellStyling = {
   fontWeight: string | undefined; // "600" | "normal" from Styling panel, but from Theme it can be any supported font-weight value
   fontStyle: string; // "italic" | "normal" from Styling panel, but from Theme it can be any supported font-weight value
   textDecoration: string; // "underline" | "none" from Styling panel, but from Theme it can be any supported font-weight value
   color: string;
   background: string;
 };
 
+export interface CellStyling extends BasicCellStyling {
```

Comment:
I've used `type` over `interface` to get a better tooltip in VSC. I think you could do the same here